src/pages/Dashboard/Dashboard.tsx

```diff
@@ -21,14 +22,19 @@ import { WalletDashboard } from './WalletDashboard'
 const direction: StackDirection = { base: 'column', xl: 'row' }
 const maxWidth = { base: 'full', lg: 'full', xl: 'sm' }
 
-const dashboardHeader = <DashboardHeader />
-
 export const Dashboard = memo(() => {
   const translate = useTranslate()
   const isDefiDashboardEnabled = useFeatureFlag('DefiDashboard')
   const { path } = useRouteMatch()
   const isNftsEnabled = useFeatureFlag('Jaypegz')
 
+  const isOpportunitiesLoading = useFetchOpportunities()
```

Comment:
Can we have the hook return `{isLoading, data}` for API consistency?

Reply:
I'll wrap in curlies for now since the data isnt used anywhere and is uuuge